### PR TITLE
complete /api/articles endpoint with TDD

### PR DESCRIPTION
**Pull request by SSheda**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -95,5 +95,43 @@ describe("GET /api/articles/:article_id", () => {
             .then(({ body }) => {
                 expect(body.msg).toBe('Path not found');
             });
-    });    
-});  
+    });
+});
+
+describe("GET /api/articles", () => {
+    test(`200: responds with an articles array of article objects, each of which should have the following
```
Comment (saimakhanom) [inline __tests__/app.test.js:102]: we should keep test description as concise as we can - this can just be 'with the correct properties'

Comment (saimakhanom) [inline __tests__/app.test.js:102]: we should have a test that checks the articles don't have a body property as this is not implicitly checked in this test (toMatchObject will ignore additional properties not specified so if there is a body property this test will not fail)

File: models/model-articles.js [hunk 2]
```diff
@@ -2,11 +2,28 @@ const db = require("../db/connection")
...
+                     LEFT JOIN comments 
+                     ON comments.article_id = articles.article_id
+                     GROUP BY articles.article_id
+                     ORDER BY articles.created_at DESC;`)
     .then((result) => {
-        if (result.rows.length === 0) {
-            return Promise.reject({ status: 404, msg: 'Path not found'})
-        }
-      return result.rows[0];
+        console.log(result.rows)
```
Comment (saimakhanom) [inline models/model-articles.js:22]: remove logs before pushing

**Comment by SSheda** [comment]
changes have been made

**Comment by saimakhanom** [comment]
lovely, all good to merge!

**Pull request merged**